kotlin/lib/src/main/kotlin/MessageAttempt.kt

```diff
@@ -0,0 +1,70 @@
+package com.svix.kotlin
+
+import com.svix.kotlin.exceptions.ApiException
+import com.svix.kotlin.internal.apis.MessageAttemptApi
+import com.svix.kotlin.models.MessageAttemptOut
+import com.svix.kotlin.models.MessageStatus
+import com.svix.kotlin.models.ListResponseEndpointMessageOut
+import com.svix.kotlin.models.ListResponseMessageAttemptEndpointOut
+import com.svix.kotlin.models.ListResponseMessageAttemptOut
+import com.svix.kotlin.models.ListResponseMessageEndpointOut
+
+class MessageAttemptListOptions(var messageStatus: MessageStatus? = null) : ListOptions() {
+	fun messageStatus(messageStatus: MessageStatus) = apply { this.messageStatus = messageStatus }
+
```

Comment:
Just to be sure, apply returns `this` so these can be chained? (sorry kotlin noob 😅)

Reply:
I'm also a noob :) yes, apply returns `this`